palisade-service/README.md

```diff
@@ -34,8 +41,7 @@ to the Filtered Resource Service and retrieve the processed request.
 
 (fields marked with * are acquired from headers metadata)
 
-The service accepts a `PalisdeRequest`, creates a token to return to the client as a `PalisadeResponse`, the Palisade Service then takes the user, resourceId and context from the `PalisadeRequest`, and sends them on to the User Service for further
-processing.
+The service accepts a `PalisdeRequest`, creates a token to return to the client as a `PalisadeResponse`, the Palisade Service then takes the user, resourceId and context from the `PalisadeRequest`, and sends them on to the User Service for further processing.
 
 ## REST Interface
 
```

Comment:
The phrase "application exposes two REST endpoints."  could do with some re-wording.  As it stands it is either ambiguous or wrong depending on the meaning of "application".  Suggest only mentioning the one service end point on the Palisade Service Web Service and then it fits with what is then discussed.  Also the PalisadeResponse has been renamed to PalisadeClientResponse and needs to be changed in the text.

Reply:
updated